akka-docs/src/main/paradox/scheduler.md

```diff
@@ -95,16 +95,16 @@ Java
 
 If you schedule functions or Runnable instances you should be extra careful
 to not close over unstable references. In practice this means not using `this`
-inside the closure in the scope of an Actor instance, not accessing `sender()` directly
+inside the closure in the scope of an Actor instance, not accessing @apidoc[sender()](actor.Actor) {scala="#sender():akka.actor.ActorRef" java="#sender()"} directly
 and not calling the methods of the Actor instance directly. If you need to
-schedule an invocation schedule a message to `self` instead (containing the
+schedule an invocation schedule a message to @apidoc[self()](actor.Actor) {scala="#self:akka.actor.ActorRef" java="#self()"} instead (containing the
 necessary parameters) and then call the method when the message is received.
 
 @@@
 
 @@@ warning
 
-All scheduled task will be executed when the `ActorSystem` is terminated, i.e.
+All scheduled task will be executed when the @apidoc[actor.ActorSystem] is terminated, i.e.
```

Comment:
There are many overloaded methods `scheduleWithFixedDelay` and `scheduleAtFixedRate` in the `Scheduler`. Which methods should I choose? Any?

Reply:
Some of them are deprecated, and some overloads are there only for Java vs Scala API (so single param list, java Duration for Java for example). I think it is fine not API-linking the methods here though if it is hard to figure out.